Formula/buildah.rb

```diff
@@ -1,28 +1,14 @@
 class Buildah < Formula
   desc "Tool that facilitates building OCI images"
   homepage "https://buildah.io"
-  url "https://github.com/containers/buildah/archive/refs/tags/v1.26.2.tar.gz"
-  sha256 "a5e95c7b50e31db80b07149cc4bd7ddbe43c2b8ef6add0f58333b250731c1168"
+  url "https://github.com/containers/buildah/archive/refs/tags/v1.26.4.tar.gz"
+  sha256 "3c29de975e8e79858a3af6278ebe5efe6140062d92abeee662b96105927d29e9"
   license "Apache-2.0"
 
   depends_on "go" => :build
   depends_on "go-md2man" => :build
   depends_on "gpgme"
 
-  # All these patches have been committed upstream, we're just cherry-picking
-  # or backporting them to v1.26.2.
-  patch do
-    url "https://raw.githubusercontent.com/slp/homebrew-krun/cd74e3bd5196134d0219d918651e6b8bbf64ead8/patches/buildah/0001-Update-vendor-of-containers-storage.patch"
```

Comment:
Indeed, this one can be dropped after updating to v1.26.4. Please also run `git rm patches/buildah/0001-Update-vendor-of-containers-storage.patch` so we drop the patch itself for the repo.